Jenkinsfile

```diff
@@ -29,7 +29,7 @@ pipeline {
           Restrict nightly builds to master branch, all others will be built on change only.
           Note: The BRANCH_NAME will only work with a multi-branch job using the github-branch-source
         */
-        cron(BRANCH_NAME == "master" ? "H H(17-19) * * *" : "")
+        pollSCM(BRANCH_NAME == "master" ? "H H(17-19) * * *" : "")
```

Comment:
Any risk with this not being built frequently, if changes are infrequent? I know with builds we only were doing pollSCM on in the past, there were concerns that "CI was not running for the project."

Reply:
i could change it to cron/weekly if that makes better sense. will it still mark it as no CI running if it runs weekly?

Reply:
changed back to nightly. We should have the discussion regarding the frequency of building apps that are rarely modified.